examples/src/main/java/com/example/ftclibexamples/PowerPlaySample/ClawSubsystem.java

```diff
@@ -0,0 +1,25 @@
+package com.example.ftclibexamples.PowerPlaySample;
+
+import com.arcrobotics.ftclib.command.Command;
+import com.arcrobotics.ftclib.command.InstantCommand;
+import com.arcrobotics.ftclib.command.SubsystemBase;
+import com.arcrobotics.ftclib.hardware.ServoEx;
+
+public class ClawSubsystem extends SubsystemBase {
+    private final ServoEx claw;
+
+    public static double grabPosition = 0.5;
+    public static double releasePosition = 0.7;
+
+    public ClawSubsystem(ServoEx claw) {
+        this.claw = claw;
+    }
+
+    public Command runGrabCommand() {
+        return new InstantCommand(() -> claw.setPosition(grabPosition), this);
+    }
+
+    public Command runReleaseCommand() {
+        return new InstantCommand(() -> claw.setPosition(releasePosition), this);
+    }
+}
```

Comment:
Add some comments to this to explain some stuff. They can be very basic. At least describe what this command does and why it has the values it has.